packages/form-js-viewer/src/Form.js

```diff
@@ -221,12 +222,14 @@ export default class Form {
 
       const fieldErrors = validator.validateField(field, value);
 
-      return set(errors, [ field.id ], fieldErrors.length ? fieldErrors : undefined);
+      return set(errors, getErrorPath(field), fieldErrors.length ? fieldErrors : undefined);
     }, /** @type {Errors} */ ({}));
 
-    this._setState({ errors });
+    const filteredErrors = this._applyConditions(errors, data, { getFilterPath: getErrorPath });
+
+    this._setState({ errors: filteredErrors });
```

Comment:
Just wondering whether this is a breaking change for `validate`, but I believe we're indeed fixing something here, the behavior beforehand was broken.

Reply:
Maybe we can add a test case for `#validate` that verifies this behavior as well?

Reply:
added it ✅